test/08_faucet_tests.ts

```diff
@@ -154,51 +148,49 @@ describe('Testing the Holograph Faucet', async () => {
 
     it('setWithdrawAmount(): Owner adjusts Withdraw Amount', async function () {
       expect(
-        await FAUCET.functions.setWithdrawAmount(DEFAULT_DRIP_AMOUNT.mul(factor))
+        await FAUCET.setWithdrawAmount(DEFAULT_DRIP_AMOUNT.mul(factor))
       ).to.not.reverted
       expect(await FAUCET.faucetDripAmount).to.equal(DEFAULT_DRIP_AMOUNT.mul(factor))
     })
 
     it('requestTokens(): User can withdraw increased amount', async function () {
-      await FAUCET.connect(USER_A).functions.requestTokens().then(tx => tx.wait())
+      await FAUCET.connect(USER_A).requestTokens().then(tx => tx.wait())
       dripCount += factor
       expect(
-        await SAMPLEERC20.functions.balanceOf(USER_A)
+        await ERC20.balanceOf(USER_A)
       ).to.equal(DEFAULT_DRIP_AMOUNT.mul(factor))
     })
 
     it('setWithdrawAmount(): Owner adjusts Withdraw Amount back to 100 eth', async function () {
       expect(
-        await FAUCET.functions.setWithdrawAmount(DEFAULT_DRIP_AMOUNT)
+        await FAUCET.setWithdrawAmount(DEFAULT_DRIP_AMOUNT)
```

Comment:
Removing all those `.functions` was unnecessary.

Reply:
Responded to this in same comment above ^^^
https://github.com/holographxyz/holograph-protocol/pull/39#discussion_r987922145